src/Uno.Templates/content/unoapp/MyExtensionsApp.1/Presentation/LoginViewModel.cs

```diff
@@ -6,18 +7,16 @@ namespace MyExtensionsApp._1.Presentation;
 
 public partial class LoginViewModel : ObservableObject
 {
-	public string Title { get; } = "Login";
+	private IAuthenticationService _authentication;
+
+	private INavigator _navigator;
 
 	[ObservableProperty]
 	private string? _username;
 
 	[ObservableProperty]
 	private string? _password;
 
-	public ICommand Login { get; }
-
-//+:cnd:noEmit
-#if useLocalization
```

Comment:
Not needed anymore?

Reply:
yep not needed, was accidentally left in